nixos/modules/services/networking/ssh/sshd.nix

```diff
@@ -272,6 +272,28 @@ in
         '';
       };
 
+      logLevel = mkOption {
+        type = types.str;
```

Comment:
I think the log levels are pretty stable,. Perhaps using types.enum is a good idea?